ui-components/__tests__/page_components/SubmitApplication.test.tsx

```diff
@@ -1,91 +1,19 @@
 import React from "react"
 import { render, cleanup } from "@testing-library/react"
-import {
-  AllFields,
-  DropOffNoOfficeHours,
-  MailingNoPostmarks,
-  MailingNoPostmarksYesDueDate,
-  MailingWithPostmarks,
-  MailingYesPostmarksNoDueDate,
-} from "../../src/page_components/listing/listing_sidebar/SubmitApplication.stories"
+import { AllFields } from "../../src/page_components/listing/listing_sidebar/SubmitApplication.stories"
 
 afterEach(cleanup)
 
-describe("<ApplicationAddresses>", () => {
+describe("<SubmitApplication>", () => {
   it("includes mailing address, includes drop off address, includes due date, includes postmarks, includes office hours", () => {
     const { getByText } = render(<AllFields />)
-    expect(getByText("Submit a Paper Application")).toBeTruthy()
-    expect(getByText("Send Application by US Mail")).toBeTruthy()
-    expect(getByText("Mailing Address Street")).toBeTruthy()
-    expect(
-      getByText(
-        "Applications must be received by the deadline. If sending by U.S. Mail, the application must be postmarked by November 29th, 2021",
-        { exact: false }
-      )
-    ).toBeTruthy()
+    expect(getByText("Paper App Header")).toBeTruthy()
+    expect(getByText("Mail Header")).toBeTruthy()
+    expect(getByText("Mailing Address Street", { exact: false })).toBeTruthy()
+    expect(getByText("Postmark details string")).toBeTruthy()
     expect(getByText("or")).toBeTruthy()
-    expect(getByText("Drop Off Application")).toBeTruthy()
-    expect(getByText("Drop Off Address Street")).toBeTruthy()
-    expect(getByText("Office Hours")).toBeTruthy()
-  })
-  it("excludes mailing address, include drop off address, excludes office hours", () => {
```

Comment:
These are now tested outside of this component in the new `postmarkString.ts` and `postmarkString.test.ts` files in `shared-helpers`, so we aren't losing coverage